examples/extract_data/src/pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[tool.poetry]
+name = "extract-data"
+version = "0.1.0"
+description = ""
```

Comment:
Add description and name here

Reply:
I can't not add the email or else poetry won't install it, I added the description